### DISCUSSION TITLE
Failure to import one meeting from a public feed

### DISCUSSION THREAD

**Discussion by aavictoria99** [code4recovery/12-step-meeting-list#1030] [Q&A]
What does the attached Google error message mean? All meetings imported successfully except this one.
![Error-1](https://user-images.githubusercontent.com/78445180/210198716-d667fde0-5ddc-42c1-ae56-3b60bf592d95.jpg)

**Comment by joshreisner** [comment]
hi, this means that the importer sees a new address but it can't look it up with google because of the referrer restriction on your API key. to fix it you could remove the restriction at [the google api console](https://console.cloud.google.com/) or switch your maps to [mapbox](https://www.mapbox.com/)

**Reply by aavictoria99** [reply]
Thank you Josh
As always you have promptly answered my query and I have fixed the problem
Andrew
